src/libraries/Microsoft.Extensions.Logging.Console/src/ConsoleLogger.cs

```diff
@@ -13,7 +14,8 @@ namespace Microsoft.Extensions.Logging.Console
     /// A logger that writes messages in the console.
     /// </summary>
     [UnsupportedOSPlatform("browser")]
-    internal sealed class ConsoleLogger : ILogger
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public sealed class ConsoleLogger : ILogger
```

Comment:
These types have a ton of breaking changes, so it might be better to just expose empty types and rename these internal ones, or not bring back any of logging.

Reply:
I wonder if the balance we might strike in logging is not to expose any of the concrete loggers or related helper types/interfaces.  Only expose the extension methods.  I might prototype this to see if that's doable.  Still looking for usage information to validate this approach.

Reply:
This is the approach I'm taking.  Only bringing back entry-point APIs and not these pub-ternal like types.